script/DeployDecoderAndSanitizer.s.sol

```diff
@@ -65,8 +64,8 @@ contract DeployDecoderAndSanitizerScript is Script, ContractNames, MainnetAddres
 
     function setUp() external {
         privateKey = vm.envUint("BORING_DEVELOPER");
-        vm.createSelectFork("mainnet");
-        setSourceChainName("mainnet"); 
+        vm.createSelectFork("bob");
```

Comment:
Does this mean the decode was initially deployed on the wrong chain, i.e., on Ethereum rather than BOB?